tests/test_units/test_metadata_class.py

```diff
@@ -127,13 +201,15 @@ def test_metadata_populate_change_access_ok(globalconfig1):
 
     edata = dio.ExportData(
         config=globalconfig1,
+        access_classification="internal",
         access_ssdl={"access_level": "paranoid", "rep_include": False},
     )
```

Comment:
Would "paranoid" be in ALLOWED_ACCESS_CLASSIFICATION"....?

Reply:
We don't currently validate anything for the access_ssdl input argument I think. Valid values according to the schema is `internal` and `asset`.